src/index.ts

```diff
@@ -40,6 +40,45 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 			return !!value.mediaType.mimeType.match('\\bjson$')
 		})
 
+		hbs.registerHelper('logger', function(value: unknown): boolean {
+			console.log(value)
+			return true
+		})
+
+		hbs.registerHelper('isAnyOperationSupportMultipart', function(operations: CodegenOperation[]): boolean {
+			return Object.values(operations).some(op => op.requestBody?.defaultContent.mediaType.mimeType === 'multipart/form-data')
+		})
+
+		hbs.registerHelper('isOperationSupportsMultipart', function(value: CodegenOperation): boolean {
+			return value.requestBody?.defaultContent.mediaType.mimeType === 'multipart/form-data'
+		})
+
+		hbs.registerHelper('isContentSupportsMultipart', function(value: CodegenContent): boolean {
```

Comment:
Is this "supports" multipart or is this it _is_ multipart? This seems like the same tests that we already have for `json`-related content.